site/_collections/algolia.js

```diff
@@ -75,16 +75,20 @@ const algoliaCollection = collections => {
       ? generateImgixSrc(item.data.hero, {w: 100, auto: 'format'})
       : '';
 
+    const url = item.data.canLinkToList
+      ? `${item.data.listUrl}#${item.fileSlug}`
```

Comment:
it feels like this logic should live in the events page rather than here. maybe events should have a property called algoliaUrl that overwrites item.url if present? Then the individual page/item itself can decide how it wants to be indexed in algolia, including redirects.

Reply:
Makes sense! Just pushing up the changes now. 

Also, It's just occurred to me that since we've introduced tabs to the page, the anchors are only going to work for items in the first tab. Ill try to address that today but wondering if we need a contingency so that we don't block the release. If we're happy with the events not showing in search initially, we could always just set isVirtualItem to false for now.

Reply:
Let's move the algolia logic from this PR and please submit an issue to cover it in next release, so that we can launch on time. Wdyt?